### PR TITLE
[ES] Add `Storyboard/Scripting/Cheat Sheet`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Storyboard/Scripting/Cheat_Sheet/es.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+En el archivo .osu, en \[Events\]: *Nota: los guiones bajos se pueden reemplazar con espacios.*
+
+### Sprites y animaciones
+
+El tamaño del campo de juego activo es de 640 píxeles de ancho por 480 píxeles de alto. Para imágenes/efectos de fondo estáticos, es mejor limitar las imágenes a estas dimensiones.
+
+Sprite estático:
+
+```
+Sprite,«capa»,«origen»,"ruta del archivo",x,y
```
[the remark this answers is not in the document]
Comment (Unnombredeusuarioextremadamentelargo) [reply wiki/Storyboard/Scripting/Cheat_Sheet/es.md:16]: Deja las comillas inglesas para que no haya confusión al momento de leer los ejemplos

File: wiki/Storyboard/Scripting/Cheat_Sheet/es.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+_evento, easing, starttime, endtime, [parámetros]
+_evento, [...]
+_evento, [...]
+```
+
+Animación:
+
+```
+Animation,«capa»,«origen»,"ruta del archivo",x,y,frameCount,frameDelay,looptype
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/Cheat_Sheet/es.md:25]:
```suggestion
Animation,"capa","origen","ruta del archivo",x,y,frameCount,frameDelay,looptype
```

File: wiki/Storyboard/Scripting/Cheat_Sheet/es.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+_evento,easing,starttime,endtime,[parámetros]
+_evento, [...]
+_evento, [...]
+```
+
+**Para animaciones, especifica un nombre de archivo como «sliderball.png», y asigna a tus archivos el nombre «sliderball0.png» a «sliderball9.png» para obtener una animación de 10 fotogramas.**
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/Cheat_Sheet/es.md:31]:
```suggestion
**Para las animaciones, especifica un nombre de archivo como «sliderball.png», y asígnale a tus archivos el nombre «sliderball0.png» a «sliderball9.png» para obtener una animación de 10 fotogramas.**
```

File: wiki/Storyboard/Scripting/Cheat_Sheet/es.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+| T | Bucle activado por un evento |
+| P | Parámetros |
+
+| Suavizado | Descripción |
+| :-: | :-- |
+| 0 | Ninguno |
+| 1 | Empezar rápido y disminuir la velocidad |
+| 2 | Empezar lento y aumentar la velocidad |
+
+**starttime, endtime**: los tiempos en milisegundos entre los cuales se animará el evento
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/Cheat_Sheet/es.md:88]:
```suggestion
**starttime, endtime**: los períodos de tiempo en milisegundos en los que se animará el evento
```

File: wiki/Storyboard/Scripting/Cheat_Sheet/es.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+**ruta del archivo**: ruta relativa del archivo de sonido (debe estar en el mismo directorio que el archivo .osu)
+
+*Nota: la ruta del archivo debe estar entre comillas dobles*
+
+**volumen**: volumen del archivo de sonido. (1-100) Se puede dejar en 100.
+
+### Variables
+
+El soporte primitivo para variables ya está disponible. Puedes definir cualquier cantidad de variables incluyendo una sección \[Variables\] en el archivo .osb (actualmente no se admite en los storyboards específicos del archivo .osu).
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Storyboard/Scripting/Cheat_Sheet/es.md:238]:
```suggestion
El soporte primitivo para las variables ya está disponible. Puedes definir cualquier cantidad de variables incluyendo una sección \[Variables\] en el archivo .osb (actualmente no se admite en los storyboards específicos del archivo .osu).
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**